src/Razor/src/Microsoft.AspNetCore.Razor.Language/BoundAttributeDescriptorBuilderExtensions.cs

```diff
@@ -52,6 +53,18 @@ public static string GetPropertyName(this BoundAttributeDescriptorBuilder builde
             builder.IndexerValueTypeName = valueTypeName;
         }
 
+        public static bool IsDirectiveAttribute(this BoundAttributeDescriptorBuilder builder)
```

Comment:
Ahhh, wrong one public. Need the BoundAttributeDescriptorExtensions.cs one public. I'll hack around it and special case the name for now.